common.yaml.jinja

```
@@ -9,6 +9,7 @@ services:
     build:
       context: ./odoo
       args:
+        DB_VERSION: "{{ postgres_version or 'latest' }}"
```

To avoid failures in case this is what makes you remove support for older versions, you can always add this argument conditionally:

```suggestion
        {%- if odoo_version >= 11 %}
        DB_VERSION: "{{ postgres_version or 'latest' }}"
        {%- endif %}
```